test/ComposerRequireCheckerTest/Cli/CheckCommandTest.php

```diff
@@ -207,4 +207,21 @@ public function testNoUnknownComposerSymbolFound(): void
             $this->commandTester->getDisplay()
         );
     }
+
+    public function testNoUnknownEnumSymbolsFound(): void
+    {
+        if (version_compare(PHP_VERSION, '8.1.0') === -1) {
```

Comment:
This should be `@requires php >= 8.1.0` (https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-requires-tables-api)

Reply:
Switched condition to annotation.